openmetadata-ui/src/main/resources/ui/src/components/ApplicationConfigProvider/ApplicationConfigProvider.tsx

```diff
@@ -66,14 +68,28 @@ const ApplicationConfigProvider: FC<ApplicationConfigProviderProps> = ({
     }
   };
 
-  const updateSelectedPersona = useCallback((persona: EntityReference) => {
-    setSelectedPersona(persona);
-  }, []);
+  const updateSelectedPersona = useCallback(
+    (persona: EntityReference) => {
+      setSelectedPersona(persona);
+    },
+    [setSelectedPersona]
+  );
 
   useEffect(() => {
     fetchApplicationConfig();
   }, []);
 
+  useEffect(() => {
+    const faviconHref = isEmpty(applicationConfig.customFaviconUrlPath)
+      ? ''
+      : applicationConfig.customFaviconUrlPath ?? '';
```

Comment:
IsEmpty will check for both empty and undefined.
```suggestion
    const faviconHref = isEmpty(applicationConfig.customFaviconUrlPath)
      ? ''
      : applicationConfig.customFaviconUrlPath;
```

Reply:
It's to make `faviconHref ` type as string instead `string | undefined`